docs/source/contributing/index.md

```diff
@@ -69,3 +66,11 @@ Make sure you add and commit the generated files in `http-examples`.
 
 conventions
 ```
+
+## Code Formatting and Testing
+
+To ensure consistent code formatting, it's recommended to have the "black" tool installed locally. You can install it by running the "make" command inside the "plone.restapi" folder. Once installed, you can use "black" to automatically format the code.
+
+The project has a set of dependencies required for its proper functioning. You can install these dependencies by running the "make" command, which will handle the installation process for you.
+
+To run tests locally and ensure your changes don't introduce any issues, use the "make" command followed by "make test". This will execute the test suite and provide test feedback.
```

Comment:
`make` itself is required only one time to install the project.

````suggestion
To run tests locally and ensure your changes don't introduce any issues, use the following command.
This will execute the test suite and provide test feedback.

```shell
make test
```
````